articles/notification-hubs/android-sdk.md

```diff
@@ -160,7 +160,7 @@ also have the connection strings that are necessary to send notifications to a d
 1. In the **build.gradle** file for the app, add the following lines in the dependencies section:
 
    ```gradle
-   implementation 'com.microsoft.azure:notification-hubs-android-sdk:1.0.0-preview1@aar'
+   implementation 'com.microsoft.azure:notification-hubs-android-sdk:1.1.4'
    implementation 'androidx.appcompat:appcompat:1.0.0'
 
    implementation 'com.google.firebase:firebase-messaging:20.1.5'
```

Comment:
Our library will pull in the necessary version of firebase. We should remove this line, and the import for `volley` below.

We actually require firebase version 21 now too, so extra important.